CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,217 @@
+
+**Contacting the Person(s) Reported**
+
+After the Quansight COC Committee has had time to review and discuss the report, any persons responsible for the incident in the report (hereafter called reportees) will be contacted. Reportees will be informed of what has been reported about them & they will be asked to provide their account of what happened. The incident reporter will not normally be consulted about the proposed resolution of the incident, except insofar as needed to understand how to help them feel safe.
+
+We aim to respond within one week to the incident reporter with either a resolution or an explanation of why the situation remains unresolved. We will contact the reportee to let them know what actions will be taken as a result of the report if any.
+
+Our policy is to make sure that everyone aware of the initial incident is also made aware that official action has been taken while still respecting the privacy of individuals. Quansight commits to maintaining the confidentiality of the reporter of an incident. Quansight may choose to make a public statement regarding the incident while maintaining the anonymity of those involved.
+
+**Response and Potential Consequences**
+
+The Quansight COC Committee will reccommend a response to the Quansight Labs Co-Directors (or higher up the Quansight leadership team if appropriate).
+
+Potential consequences (for reportees) of Quansight Code of Conduct incidents include:
+
+* Nothing (if it is determined that no breach occurred)
+* Private feedback or reprimand from Quansight to the reportee(s) involved
+* Warning the reportee to cease their behavior and that any further incident reports will result in sanctions
+* A public announcement that an incident occurred
+* Mediation (only if both reporter(s) and reportee(s) agree)
+* A permanent or temporary ban from some or all Quansight spaces (mailing lists, GitHub repos, in-person events, etc.)
+* Assistance to the complainant with a report to other bodies (e.g., institutional offices or appropriate law enforcement agencies)
+* An imposed vacation or removal from Quansight employment/affiliation (for Quansight personnel/affiliates)
+* Publishing an account of the incident and calling for the resignation of the reportee(s) from their responsibilities (usually pursued when Quansight lacks formal authority: this may be appropriate if the person is an external event leader, if they refuse to recuse themselves from a conflict of interest, or similar)
+* Any other response that Quansight deems necessary and appropriate to the situation
+* At Quansight events, if a participant engages in behavior that violates this Code of Conduct, the organizers and staff may take any action they deem appropriate
+
+At an in-person company event, potential consequences for Code of Conduct incidents include:
```

Comment:
This is kind of in conflict with the purpose of this COC. First sentence of the introduction: _"https://numpy.org/neps/nep-0023-backwards-compatibility.html#policy"_

For "company events" I first think of something like the retreat or other in-person business-related meetings or social events, for which this COC does not apply (we have an employee handbook etc.). If it's, say, a workshop for a Quansight-driven open source project, this COC should apply. So to resolve this I think:

- Keep an in-person part, but change wording from "company event" to Quansight-organized event"
- Fix the first sentence of the Introduction to include such events